lib/charms/layer/containerd.py

```diff
@@ -19,6 +19,15 @@ class ResourceFailure(Exception):
     pass
 
 
+def can_mount_cgroup2() -> bool:
+    """Determine if it's possible to mount cgroup2 type filesystems."""
+    try:
+        stdout = check_output(["mount", "-t", "cgroup2"], text=True)
+    except CalledProcessError:
+        return False
```

Comment:
I would log the exception here so we have visibility when something goes wrong. Maybe something like `hookenv.log(traceback.format_exc(), level=hookenv.ERROR)`

Reply:
👍🏻 gotcha covered!